controllers/web.js

```diff
@@ -2,53 +2,62 @@
 
 const web = require("../platforms/web/web");
 const bot = require("../bots/hosting")();
-const config = require("../config/config-loader").load();
 const apiai = require("../utils/apiai");
 const Bluebird = require("bluebird");
-const { ButtonsListMessage, Button} = require("../platforms/generics");
-const uuid = require("uuid/v4");
 
 module.exports = () => {
-
-  function postbackReceived(res, senderId, payload) {
-    return bot.ask("postback", senderId, payload, null, null, res)
-      .then((answer) => {
+  function postbackReceived(res, nichandle, payload) {
+    return bot
+      .ask("postback", nichandle, payload, null, null, res)
```

Comment:
actualy if you look at the doc it's is written this way :)
[https://eu.api.ovh.com/console/#/me#GET](https://eu.api.ovh.com/console/#/me#GET)